src/utils/utils.py

```diff
@@ -39,6 +39,7 @@ def get_data(self, sample):
         if self.config["experiment_type"] == "attack":
             items["z"] = Variable(sample["z"]).to(self.device)
             items["x"] = Variable(sample["x"]).to(self.device)
+            items["img"] = Variable(sample["img"]).to(self.device)
```

Comment:
check if redundant

Reply:
This is used in evaluating the input optimization model, by comparing the reconstructed input to the original image.